lib/utils.js

```diff
@@ -258,6 +258,19 @@ exports.autoStoreData = function (req, res, next) {
     req.session.data = {}
   }
 
+  try {
+    let defaultsFilePath = path.join(__dirname, '/../app/data/session-defaults.js')
+
+    if (fs.existsSync(defaultsFilePath)) {
+      let defaults = require(defaultsFilePath)
+      defaults = Object.assign(defaults, req.session.data)
+
+      storeData(defaults, req.session)
+    }
+  } catch (e) {
+    console.error('Could not load defaults file')
```

Comment:
Could this be more specific about what the defaults file is? Default user data file?

Reply:
It could also specify the location it's looking for.